installation/supported-software.md

```diff
@@ -0,0 +1,6 @@
+# Supported software
+
+| medic-webapp | Node | CouchDB | Browsers | SMSSync | medic-transport | Android | medic-android |
+|----|----|----|----|----|----|----|----|
+| **0.4** | 0.12+ | 1.6 | Chrome 30+, Firefox latest | Yes | No | N/A | N/A |
+| **2.x** | 6+ | 1.6 | Chrome 30+, Firefox latest | No | Yes | 4.4+ | Any |
```

Comment:
Specifically we support node 6.10. I don't know how specific you want to be though. I don't know what changes there are between 6.0 and 6.10, and whether it would actually matter.